src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -503,6 +503,29 @@ <h2 style="color: white">Passende Kompetenzen</h2>
 	</div>
 
 	<!-- Criteria Panel -->
+	<div class="formsection" style="border-radius: 25px; background-color: #f1f0ff">
+		<div class="formsection-x-title">
+			<h2 class="u-text-dark2">Dauer</h2>
+		</div>
+		<div class="formsection-x-help">
+			<h3 class="u-text-body-bold">Was ist die Dauer?</h3>
+			<p class="u-text u-margin-top1x u-margin-bottom2x">
+				Im Dauerfeld beschreibst Du den zeitlichen Umfang der Leistung, die nötig ist, um das Badge zu erhalten.
```

Comment:
A note that it's optional and can be left at "0", to disable it, would be nice

Reply:
Marked the field as optional